### PR TITLE
Sample orbits

### LINKED ISSUE #237: Add method to sample an orbit
This is very useful for things like generating a table of elements (or `DataFrame`s, @Guillem-db!). One option would be receiving an array of `Time` values and returning a list of `Orbit` objects, but I do not have a strong opinion about this and it is open to discussion. Other options are:

* A `CartesianRepresentation`-like object, which can contain an array of values.
* An "interpolator-like" object - but this would be merely a proxy to `propagate`
* ?

Also, bear in mind that we should ensure that propagating backwards works.

### PR DESCRIPTION
**Pull request by Juanlu001**
Fix #237.

### CODE REVIEW

**Comment by Juanlu001** [comment]
Floating point issues when manipulating astropy times strike back: https://github.com/astropy/astropy/issues/6638

**Comment by Juanlu001** [comment]
I like how this is evolving :)

screenshot-2017-10-12 untitled

**Comment by Juanlu001** [comment]
This pull request is ready for review.

**Comment by Juanlu001** [comment]
This, by the way, unblocks #218 and #117.

**Comment by anhiga** [comment]
I'll review this as soon as possible. By the way, Travis keep failing due to notebook execution.

**Comment by Juanlu001** [comment]
You are right about the notebooks... I restarted the job and failed nonetheless. It's always the same notebook, so I'm going to disable it.

File: src/poliastro/util.py [hunk 1]
```diff
@@ -71,3 +72,22 @@ def norm(vec):
 
     """
     return np.sqrt(vec.dot(vec))
+
+
+def time_range(start, *, periods=50, spacing=None, end=None):
+    """Generates range of astronomical times.
```
Comment (anhiga) [inline src/poliastro/util.py]: This docstring should have `Parameters` and `Returns`.

Comment (Juanlu001) [reply src/poliastro/util.py]: Good point, thanks! Will do.

**Comment by Juanlu001** [comment]
Thanks @anhiga, as always! I improved the docstring as you suggested and did other minor doc additions. Will merge when the build is green.

**Pull request merged**